app/index.js

```diff
@@ -14,6 +15,37 @@ var paths = {
   'gulpfile.js': {}
 };
 
+function logWarning(message) {
+  var warningSign = clc.bold.cyan('<' + clc.bold.redBright('!') + '> ');
+  console.log(warningSign + clc.cyanBright(message));
+}
+
+function outputWarning() {
```

Comment:
improve name of method.  I was trying to figure out where the warning was coming from, until I read further on.